main.qml

```diff
@@ -930,12 +924,19 @@ ApplicationWindow {
                     + ", fee: " + walletManager.displayAmount(transaction.fee));
 
             // here we show confirmation popup;
+            var transactionAmount = Number(walletManager.displayAmount(transaction.amount));
+            var transactionFee = Number(walletManager.displayAmount(transaction.fee));
 
-            transactionConfirmationPopup.title = qsTr("Confirmation") + translationManager.emptyString
-            transactionConfirmationPopup.text  = qsTr("Please confirm transaction:\n")
-                        + qsTr("\n\nAmount: ") + walletManager.displayAmount(transaction.amount)
-                        + qsTr("\nFee: ") + walletManager.displayAmount(transaction.fee)
-                        + translationManager.emptyString
+            function showFiatConversion(valueXMR) {
+                return (fiatApiConvertToFiat(valueXMR) === "0.00" ? "(<0.01 " + fiatApiCurrencySymbol() + ")" : "(~" + fiatApiConvertToFiat(valueXMR) + " " + fiatApiCurrencySymbol() + ")");
+            }
+            
+            transactionConfirmationPopup.title = qsTr("Confirm transaction") + translationManager.emptyString
+            transactionConfirmationPopup.text =  "";
+            transactionConfirmationPopup.text += qsTr("Please confirm transaction to sweep unmixable outputs") + ":\n\n"
+                                              +  qsTr("Amount") + ":\n" + transactionAmount + " XMR " + (persistentSettings.fiatPriceEnabled ? showFiatConversion(transactionAmount) : "") + "\n\n"
+                                              +  qsTr("Fee") + ":\n" + transactionFee + " XMR " + (persistentSettings.fiatPriceEnabled ? showFiatConversion(transactionFee) : "")
+                                              +  translationManager.emptyString
             transactionConfirmationPopup.icon = StandardIcon.Question
```

Comment:
Seems we don't need this anymore

Reply:
Do you mean we should just remove this lines and have a cleaner transactionConfirmationPopup, 
or that the transactionConfirmationPopup is duplicated in the code and we should call `onTransactionCreated() `from `handleSweepUnmixable`?

Reply:
Was refferring to `transactionConfirmationPopup.icon = StandardIcon.Question` line, you already set the icon in `transactionConfirmationPopup` properties.